File: src/jvmMain/java/pgsql/ctl/IPgSqlCtl.java
package pgsql.ctl;

/* Simone 08/07/2014 11:54 */

public interface IPgSqlCtl {

    CtlStatus status();

    StopStatus stop();
    StopStatus stopFast();

    void start();

    void initDb();

    boolean dbExist();

}

File: satd-classifier/src/jvmMain/kotlin/pgsql/ctl/IPgSqlCtl.kt
package pgsql.ctl

/* Simone 08/07/2014 11:54 */
interface IPgSqlCtl {
    fun status(): CtlStatus
    fun stop(): StopStatus
    fun stopFast(): StopStatus
    fun start()
    fun initDb()
    fun dbExist(): Boolean
}